src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -63,7 +72,7 @@
     private final EditPersonDescriptor editPersonDescriptor;
 
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
```

Comment:
The description should be a sentence on its own. When compiled into javadocs, the description is separated from the parameter name.